run-build-functions.sh

```diff
@@ -323,6 +332,24 @@ install_dependencies() {
       echo "Error installing pip dependencies"
       exit 1
     fi
+  elif [ -f Pipfile ]
+  then
+    echo "Installing dependencies from Pipfile"
+    if $HOME/python$PIPENV_RUNTIME/bin/pipenv install
+    then
+      echo "Pipenv dependencies installed"
+      if source $($HOME/python$PIPENV_RUNTIME/bin/pipenv --venv)/bin/activate
+      then
+        echo "Python version set to $(python -V)"
+      else
+        echo "Error activating Pipenv environment"
+        echo "Please see https://github.com/netlify/build-image/#included-software for current versions"
```

Comment:
this isnt particularly useful here, as they would go to that page to learn that "Latest version." is the current version, and they would find the current version was actually in the log a few lines above this. ;-)

Reply:
Well, I thought this refers to list of python versions user can use.